spec.html

```diff
@@ -33789,6 +33789,7 @@ <h1>Array.prototype.forEach ( _callbackfn_ [ , _thisArg_ ] )</h1>
           <p>If a _thisArg_ parameter is provided, it will be used as the *this* value for each invocation of _callbackfn_. If it is not provided, *undefined* is used instead.</p>
           <p>_callbackfn_ is called with three arguments: the value of the element, the index of the element, and the object being traversed.</p>
           <p>`forEach` does not directly mutate the object on which it is called but the object may be mutated by the calls to _callbackfn_.</p>
+          <p>The range of elements processed by `forEach` is set before the first call to _callbackfn_. Elements which are appended to the array after the call to `forEach` begins will not be visited by _callbackfn_. If existing elements of the array are changed, their value as passed to _callbackfn_ will be the value at the time `forEach` visits them; elements that are deleted after the call to `forEach` begins and before being visited are not visited.</p>
```

Comment:
This "elements that are deleted after the call" portion is missing from `find`/`findIndex`. Should we also fix that?

Reply:
👍 I'll add that.

Reply:
Done now